sdk/inc/azure/core/internal/az_http_internal.h

```diff
@@ -11,6 +11,12 @@
 
 #include <azure/core/_az_cfg_prefix.h>
 
+enum
+{
+  /// The maximum number of HTTP pipeline policies allowed.
+  _az_MAXIMUM_NUMBER_OF_POLICIES = 10,
```

Comment:
We follow this pattern elsewhere already.

The guidelines suggest to use enums over macro and const:
https://azure.github.io/azure-sdk/clang_implementation.html#clang-no-magic-numbers
> Prefer enum values since the debugger can display the label and value and no memory is allocated. If you use const, memory is allocated. If you use #define, the debugger cannot display the label.